ticdc/manage-ticdc.md

```diff
@@ -749,7 +749,9 @@ To create a cyclic replication task, take the following steps:
 + The name of the table with cyclic replication enabled must match the `^[a-zA-Z0-9_]+$` regular expression.
 + Before creating the cyclic replication task, the tables for the task must be created.
 + After enabling the cyclic replication, you cannot create a table that will be replicated by the cyclic replication task.
++ To avoid causing errors, please do not execute DDL operations such as `ADD COLUMN`/`DROP COLUMN` when multiple clusters are written at the same time.
```

Comment:
```suggestion
+ To avoid causing errors, do not execute DDL statements such as `ADD COLUMN`/`DROP COLUMN` when data is written into multiple clusters at the same time.
```